### PR TITLE
Remove excess padding around suggestion items

### PR DESCRIPTION
**Pull request by j-f1**
~changes from this PR are in b6d064603b5ca95561c621335d5361351eb9df3a, and this should only be merged after #2923~

Before/after:
Screenshot_2022-02-22 09 04 16Screenshot_2022-02-22 09 03 15

If the additional padding is desired, I’m happy to add it back.

### CODE REVIEW

File: Shared/UI/Util/ButtonStyles.swift [hunk 1]
```diff
@@ -16,6 +16,7 @@ public struct TableCellButtonStyle: ButtonStyle {
                 Color.selectedCell
                     .opacity(configuration.isPressed ? 1 : 0)
                     .padding(padding)
+                    .ignoresSafeArea()
```
Comment (evancrow) [inline Shared/UI/Util/ButtonStyles.swift:19]: Can you make this:
```suggestion
                    .ignoresSafeArea(.container)
```

Recently have had a lot of issues with keyboards and `ignoreSafeArea`

**Review by evancrow — APPROVED** [review]
Padding changes LGTM

**Comment by j-f1** [comment]
ok, this should be good to merge!

**Comment by evancrow** [comment]
@darinwf Should probably be possibly for outside contributors to merge on their own, once the PR is approved. Also, tests don't seem to appear on PRs from outside contributors.

**Pull request merged**